### ISSUE TITLE
cov-core: uninstall leaves pth file behind

### ISSUE THREAD

**Issue by asottile** [schlamar/pytest-cov#53]
```
$ virtualenv venv
$ . venv/bin/activate
$ pip install cov-core
$ pip uninstall cov-core
Uninstalling cov-core:
  /tmp/venv/lib/python2.7/site-packages/cov_core-1.15.0-py2.7.egg-info
  /tmp/venv/lib/python2.7/site-packages/cov_core.py
  /tmp/venv/lib/python2.7/site-packages/cov_core.pyc
  /tmp/venv/lib/python2.7/site-packages/cov_core_init.py
  /tmp/venv/lib/python2.7/site-packages/cov_core_init.pyc
Proceed (y/n)? y
  Successfully uninstalled cov-core
$ ls venv/lib/python2.7/site-packages/init_cov_core.pth
venv/lib/python2.7/site-packages/init_cov_core.pth
```

**Comment by schlamar** [comment]
Yes, this is a known issue. See uninstall instructions: https://github.com/schlamar/pytest-cov/blob/2.0/pytest-cov/README.rst#uninstallation

This shouldn't be such an issue when https://github.com/schlamar/pytest-cov/commit/ee119a062af6cc3171bc7bc5c441f24137838fc4 gets released.

However, AFAIS @bukzor has already mentioned a solution to this issue, but I can't find it anymore :(

**Comment by asottile** [comment]
This is the approach buck and I have used with success: https://github.com/ionelmc/python-hunter/blob/c0ea3e0f02b481bafb49bbcaf36b0aa5e2a93e8d/setup.py

**Comment by schlamar** [comment]
Will be fixed in 2.0 (PR #58)
